src/homepageExperience/utils.ts

```diff
@@ -15,19 +15,19 @@ export const HOMEPAGE_NAVIGATION_STEPS = [
   },
   {
     name: 'Initialize\nClient',
-    glyph: IconFont.FileSettings,
+    glyph: IconFont.CogSolid_New,
```

Comment:
Can you remind what the rationale for replacing `FileSettings` with `CogSolid` is? Was that also a cog before, or was it something else? I can't see the `FileSettings` icon in storybook.